src/default/assets/js/src/typedoc/components/Filter.ts

```diff
@@ -81,38 +83,57 @@ namespace typedoc
 
     class FilterItemSelect extends FilterItem<string>
     {
-        private $select!:JQuery;
+        private select!:HTMLElement;
 
 
         protected initialize() {
-            $html.addClass('toggle-' + this.key + this.value);
-
-            this.$select = $('#tsd-filter-' + this.key);
-            this.$select.on(pointerDown + ' mouseover', () => {
-                this.$select.addClass('active');
-            }).on('mouseleave', () => {
-                this.$select.removeClass('active');
-            }).on(pointerUp, 'li', (e) => {
-                this.$select.removeClass('active');
-                this.setValue(($(e.target).attr('data-value') || '').toString());
+            document.documentElement.classList.add('toggle-' + this.key + this.value);
+
+            const select = document.querySelector<HTMLElement>('#tsd-filter-' + this.key);
+            if (!select) return;
+
+            this.select = select;
+            const onActivate = () => {
+                this.select.classList.add('active');
+            };
+            const onDeactivate = () => {
+                this.select.classList.remove('active');
+            };
+
+            this.select.addEventListener(pointerDown, onActivate);
+            this.select.addEventListener('mouseover', onActivate);
+            this.select.addEventListener('mouseleave', onDeactivate);
+
+            this.select.querySelectorAll('li').forEach(el => {
+                el.addEventListener(pointerUp, (e) => {
+                    select.classList.remove('active');
+                    this.setValue((e.target as HTMLElement).dataset.value || '');
+                })
             });
 
-            $document.on(pointerDown, (e) => {
-                var $path = $(e.target).parents().addBack();
-                if ($path.is(this.$select)) return;
+            document.addEventListener(pointerDown, (e) => {
+                if (this.select.contains(e.target as HTMLElement)) return;
```

Comment:
Neat, I don't think I've ever used contains on a dom node before.

Reply:
I think it is the same. In this code we have found all parents:
```ts
$(e.target).parents().addBack()
```
And we check that `select` is parent for `e.target`: 
```ts
$path.is(this.$select)
```
So, `this.select.contains(e.target as HTMLElement)` should works same way. I have read, all compatibility table is green [here](https://developer.mozilla.org/en-US/docs/Web/API/Node/contains). I have checked, `this.select.contains(this.select)` returns `true`, so this covers this case `.addBack()`. I have debugged it via mobile emulator in Chrome,  and it seems it works correctly.